docs/spec.md

```diff
@@ -3317,25 +3527,31 @@ and will likely be removed in the future
 
 #### Inputs
 
-| Label | Name          | Type                                         | Constraints |
-|-------|---------------|----------------------------------------------|-------------|
-| (I1)  | `inputs`      | variadic number of tensors                   | (C1-C4)     |
-| (I2)  | `dimensions`  | 1-dimensional tensor constant of type `si64` | (C3)        |
-| (I3)  | `computation` | function                                     | (C4)        |
+| Label | Name          | Type                                            | Constraints |
+|-------|---------------|-------------------------------------------------|-------------|
+| (I1)  | `inputs`      | variadic number of tensors or quantized tensors | (C1-C6)     |
+| (I2)  | `dimensions`  | 1-dimensional tensor constant of type `si64`    | (C3)        |
+| (I3)  | `computation` | function                                        | (C4), (C6)  |
 
 #### Outputs
 
-| Name     | Type   | Constraints |
-|----------|--------|-------------|
-| `result` | tensor | (C1), (C4)  |
+| Name     | Type                       | Constraints      |
+|----------|----------------------------|------------------|
+| `result` | tensor or quantized tensor | (C1), (C4), (C6) |
 
 #### Constraints
 
 * (C1) All `inputs` and `result` have the same shape.
 * (C2) size(`inputs`) $=$ N $\ge$ 1.
 * (C3) `dimensions = [0, ..., R-1]`, where `R` $=$ rank(`inputs[0]`).
-* (C4) `computation` has type `(tensor<E0>, ..., tensor<EN-1>) -> tensor<E'>`
-  where `Ek` $=$ element_type(`inputs[k]`) and `E'` $=$ element_type(`result`).
+* If the operation uses non-quantized tensors:
+  * (C4) `computation` has type `(tensor<E0>, ..., tensor<EN-1>) -> tensor<E'>`
+     where `Ek` $=$ element_type(`inputs[k]`) and `E'` $=$ element_type(`result`).
+* If the operation uses quantized tensors:
+  * (C5) is_quantized_tensor(inputs[i]) for all `i in range(size(inputs))`.
+  * (C6) `computation` has type `(tensor<E0>, ..., tensor<EN-1>) -> tensor<E'>`
+     where `Ek` $=$ quantized_element_type(`inputs[k]`) and `E'` $=$
+     quantized_element_type(`result`).
```

Comment:
Can we keep C4 as is? The semantics of `map` would then be: `computation` is a function that takes whatever and returns whatever, and map simply applies it elementwise to inputs and write the result to the output.

Reply:
That would be great!
The only concern I had here is that  the member name function  `element_type` returns the `ElementType` part of a `TensorType`.  For `QuantizedTensorType`, I am using the member name function `quantized_element_type`. Also, I note that `baseline_element_type` may not be suitable here if we just want to use `C4` as is. 

Please let me know what do you think.